internal/codeintel/uploads/background/expiration/init.go

```diff
@@ -12,5 +15,11 @@ func NewExpirer(uploadSvc UploadService, policySvc PolicyService, policyMatcher
 		policySvc:     policySvc,
 		policyMatcher: policyMatcher,
 		metrics:       metrics,
+		logger: log.Scoped("NewExpirer", "").With(
```

Comment:
I think we've mentioned similar feedback before, but this logger will be logging things for an expirer, not just the function NewExpirer, so we don't want to scope it to NewExpirer

```suggestion
		logger: log.Scoped("Expirer", "").With(
```